source/_components/media_player.vizio.markdown

```diff
@@ -70,34 +86,40 @@ media_player:
   - platform: vizio
     host: IP_ADDRESS
     access_token: AUTH_TOKEN
+    device_type: tv
 ```
 
 {% configuration %}
 host:
-  description: IP address of your TV.
+  description: IP address of your device.
   required: true
   type: string
 access_token:
   description: Authentication token you received in the last step of the pairing process.
-  required: true
+  required: false
+  type: string
+device_class:
```

Comment:
`device_type` or `device_class`?

Reply:
device_class because I saw CONF_DEVICE_CLASS was available as a standard option. I would be OK with changing this to device_type (it would be a conf type) if you think that makes more sense.

Reply:
This depends on which variable name it is given in the python code.

Reply:
Understood. I am using device_class as verified in my test HA configuration